### ISSUE TITLE
bundled chai module fails due to missing file

### ISSUE THREAD

**Issue by adamstallard** [arangodb/arangodb-docker#41]
version 3.2.9 (and 3.2.8) have this issue.  I haven't tried other versions.

If you `require('chai')` in a foxx test file you will get this error in the arango logs
 
```
at Object. (/usr/share/arangodb3/js/node/node_modules/chai/index.js:1:272)
at require (/usr/share/arangodb3/js/common/bootstrap/modules.js:95:21)
at Function.Module._load (/usr/share/arangodb3/js/common/bootstrap/modules.js:484:16)
at Module.load (/usr/share/arangodb3/js/common/bootstrap/modules.js:539:36)
at Object.Module._extensions..js (/usr/share/arangodb3/js/common/bootstrap/modules.js:616:12)
at Module._compile (/usr/share/arangodb3/js/common/bootstrap/modules.js:598:8)
at Object. (/usr/share/arangodb3/js/node/node_modules/chai/lib/chai.js:78:14)
at require (/usr/share/arangodb3/js/common/bootstrap/modules.js:95:21)
at Function.Module._load (/usr/share/arangodb3/js/common/bootstrap/modules.js:439:27)
at Function.Module._resolveFilename (/usr/share/arangodb3/js/common/bootstrap/modules.js:520:13)

File: ./chai/interface/expect
```

The file referenced in `chai/lib/chai.js:78` (./chai/interface/expect) is missing:

```
root@404a36229e10:/usr/share/arangodb3/js/node/node_modules/chai/lib/chai/interface# ls
assert.js  should.js
```

there is no expect.js

**Comment by dothebart** [comment]
Hi, we have fixed this with the release of ArangoDB 3.3.8; once https://github.com/docker-library/official-images/pull/4300#issuecomment-385548603 has been merged it will be available on the official docker library image; the `arangodb/arangodb` image is already available. 

Closing this now.

**Comment by dothebart** [comment]
@adamstallard the docker library is now up to date too.
